website/www/site/static/.htaccess

```diff
@@ -20,7 +20,5 @@ RewriteRule ^(.*)$ https://beam.apache.org/$1 [L,R=301]
 # path /documentation/sdks/(javadoc|pydoc)/..
 # The following redirect maintains the previously supported URLs.
 RedirectMatch permanent "/documentation/sdks/(javadoc|pydoc)(.*)" "https://beam.apache.org/releases/$1$2"
-# Keep this updated to point to the current release.
-RedirectMatch "/releases/([^/]+)/current(.*)" "https://beam.apache.org/releases/$1/2.34.0$2"
```

Comment:
Would not this break the current docs? I assume we first need to generate a pydoc/current and then we can remove this.

Reply:
You're right. I'll skip this line until /current is generated, is that ok?